### DISCUSSION TITLE
Issue while creating ONLINE indexes for localhost.

### DISCUSSION THREAD

**Discussion by anuksap** [cake-build/cake#4327] [Extension Q&A] [answered]
Hello Team,
I was trying to migrate from rh to grate.
So I executed grate command locally to see changes on localhost . But it got failed for online indexes.
When I did ONLINE = OFF it works. But rh works totally fine for Online = ON.
Could you please assist if anything I am missing.

Below are the snippets from logs
Online index operations can only be performed in Enterprise edition of SQL Server or Azure SQL Edge.

**Comment by patriksvensson** [comment, marked as answer]
You've written the question in the wrong repository...
